File: src/Components/app.story.lua
local TS = require(game:GetService("ReplicatedStorage"):WaitForChild("include"):WaitForChild("RuntimeLib"));
local Roact = TS.import(script, TS.getModule(script, "roact").roact.src);

return function(target)
    local App = require(script.Parent.app)

    local element = Roact.createElement(App.default, {},{})
    local mount = Roact.mount(element, target)

    return function()
        Roact.unmount(mount)
    end
end

File: src/Components/app.story.ts
import Roact from "@rbxts/roact";
import app from "./app";

/**
 * Remember when hoarcekatting you gotta edit RuntimeLib
 * so it doesn't yield for game to load
 * turn the invalid module access error to warning
 **/

export = (target: GuiObject) => {
	const element = Roact.createElement(app, {});

	const mount = Roact.mount(element, target, "app");

	return () => {
		Roact.unmount(mount);
	};
};
